Dockerfile

```diff
@@ -23,23 +23,21 @@ RUN cd ./modules/oas-page-builder \
 FROM ubuntu:20.04
 ARG SNOOTY_PARSER_VERSION=0.13.16
 ARG SNOOTY_FRONTEND_VERSION=0.13.36
-# The Redoc CLI branch will most likely stay static. Updates to the branch should 
+# The Redoc CLI branch will most likely stay static. Updates to the branch should
 # be limited to CLI bug fixes and Redoc dependency version bumps
 ARG REDOC_CLI_VERSION=0.1.1
 ARG FLIT_VERSION=3.0.0
```

Comment:
Is this arg safe to remove as well since we're no longer using flit?

Reply:
OOOH yes, good call. Thanks for the catch!